frontend/app/src/types/settings/accounting.ts

```diff
@@ -0,0 +1,59 @@
+import { z } from 'zod';
+import { CollectionCommonFields } from '@/types/collection';
+import { type PaginationRequestPayload } from '@/types/common';
+
+export enum AccountingMethod {
+  SPEND = 'spend',
+  ACQUISITION = 'acquisition'
+}
+
+const AccountingMethodEnum = z.nativeEnum(AccountingMethod);
+
+export enum AccountingTreatment {
+  SWAP = 'swap',
+  SWAP_WITH_FEE = 'swap with fee'
+}
```

Comment:
Don't we get these dynamically somehow by the backend?

Reply:
no we don't get it from backend. It just stated on the docs